shared/storage.ts

```diff
@@ -208,13 +222,25 @@ class Storage {
   private static async loadVCs(completeBackupData: any, encryptionKey: any) {
     const allVCs = completeBackupData['VC_Records'];
     const allVCKeys = Object.keys(allVCs);
+    const dataFromDB = completeBackupData['dataFromDB'];
+
     allVCKeys.forEach(async key => {
-      const vc = allVCs[key];
+      let vc = allVCs[key];
+      const currentUnixTimeStamp = Date.now();
+      const prevUnixTimeStamp = vc.vcMetadata.timestamp;
+      vc.vcMetadata.timestamp = currentUnixTimeStamp;
+      dataFromDB.myVCs.forEach(myVcMetadata => {
+        if (
+          myVcMetadata.requestId === vc.vcMetadata.requestId &&
+          myVcMetadata.timestamp === prevUnixTimeStamp
+        ) {
+          myVcMetadata.timestamp = currentUnixTimeStamp;
```

Comment:
This codechange will not have a side-effect as this is being assigned to the loop variable `myVcMetadata`, 
 so this change will not reflect back to the `allVCs array`. I'll make this change in my PR.

Reply:
I was wrong about the above yesterday as for objects, the above doesn't hold true and it gets modified.

```
> let a = [{c: 1, d: 3, e: "alice"}, {c: 2, d: 4, e: "bob"}]
undefined
> a.forEach(x => x.c+=1)
undefined
> a
[ { c: 2, d: 3, e: 'alice' }, { c: 3, d: 4, e: 'bob' } ]
```
for numbers it's not
```
> let c = [1, 2, 3]
undefined
> c.forEach(x => x+=1)
undefined
> c
[ 1, 2, 3 ]
```